section2/exercises/ex3_bonus.rb

```diff
@@ -0,0 +1,47 @@
+puts "You need to pick out your outfit for the day. Do you want to (1) look fancy or (2) dress comfortably?"
+
+print "> "
+choice = $stdin.gets.chomp
+
+if choice == "1"
+  puts "1. Black-tie"
+  puts "2. Business"
+  puts "3. Business Casual"
+  puts "4. Smart Casual"
+
+  print "> "
+  fancy = $stdin.gets.chomp
+
+  if fancy == "1"
+    puts "Where do you even buy something like this? Good luck!"
+  elsif fancy == "2"
+    puts "Are you applying for a loan? Good luck!"
+  elsif fancy == "3"
+    puts "Perfect, you can hit the golf course after work. Good luck!"
+  elsif fancy == "4"
+    puts "What even is smart casual? You'd better start Googling. Good Luck!"
```

Comment:
Hahhaah! This is what I selected when I ran your file. I was pleasantly surprised :)